.github/workflows/release.yml

```
@@ -21,18 +21,18 @@ jobs:
         with:
           # Allow goreleaser to access older tag information.
           fetch-depth: 0
-      - uses: actions/setup-go@v5.2.0
+      - uses: actions/setup-go@v5.5.5
```

```suggestion
      - uses: actions/setup-go@v5.5.0
```

5.5.0 is the latest stable release for this action